prototype/frameworks/llamastack/kubernetes/llama-stack/deployment.yaml

```diff
@@ -25,6 +25,10 @@ spec:
           value: http://vllm:8000/v1
         - name: VLLM_API_TOKEN
           value: fake
+        - name: GRANITE_URL
+          value: http://granite:8000/v1
```

Comment:
@cooktheryan so would this llama stack deployment server have two models configured and the user can choose which one to use? or would a separate deployment be created for a new model?

Reply:
the first one. This allows the user to choose between using the llama model and the granite model depending on their application use case

Reply:
makes sense, thank you!